src/components/Home/UpgradeNotice.js

```diff
@@ -7,22 +7,45 @@ import {
   selectCanClusterUpgrade,
   selectLoadingFlagByAction,
 } from 'selectors/clusterSelectors';
-import { UpgradeNoticeStyles } from 'styles';
 import LoadingOverlay from 'UI/LoadingOverlay';
 
 const UpgradeWrapperDiv = styled.div`
   display: inline-block;
-  ${UpgradeNoticeStyles({ textDecoration: 'none' })}
+  color: ${props => props.theme.colors.gold};
+  cursor: ${({ hasFunction }) => (hasFunction ? 'pointer' : 'inherit')};
```

Comment:
Why not use the `onClick` prop for this check?

Reply:
good catch!